riscv/mmu.h

```diff
@@ -126,6 +126,16 @@ class mmu_t
       throw trap_store_guest_page_fault(t.get_tval(), t.get_tval2(), t.get_tinst()); \
     }
 
+  #define convert_trigger_before_to_trigger_after(BODY) \
```

Comment:
I see no benefit to the macro here. It's only used once, and BODY is only used once.

Reply:
Attempt to legalize tdata1.timing instead of converting behavior on firing.